scout/server/blueprints/variants/templates/variants/sv-variant.html

```diff
@@ -223,13 +228,24 @@
 
   <div class="row">
     <div class="col-md-12">
-      {{ genes(variant) }}
+      {{ overlapping(overlapping_snvs, variant.rank_score) }}
     </div>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      {{ overlapping(overlapping_snvs, variant.rank_score) }}
+    <div class="col-12">
+      <div class="card">
+        <nav>
+          <div class="nav nav-tabs" id="nav-tab" role="tablist">
+            <a class="nav-item nav-link active" id="nav-genes-tab" data-toggle="tab" href="#nav-genes" role="tab" aria-controls="nav-genes" aria-selected="true">Genes</a>
+            <a class="nav-item nav-link" id="nav-transcripts-tab" data-toggle="tab" href="#nav-transcripts" role="tab" aria-controls="nav-transcripts" aria-selected="false">Transcripts</a>
+          </div>
+        </nav>
+        <div class="tab-content" id="nav-tabContent">
```

Comment:
Maybe you could add mt-3 to this div class, to distantiate the a bit the div from the top?
div class="tab-content mt-3"

Reply:
:+1: